File: Dragon_Dungeons.client/src/models/Account.js
import { RepoItem } from "./RepoItem.js"

export class Profile extends RepoItem {
  constructor(data) {
    super(data)
    this.name = data.name
    this.picture = data.picture
  }
}

export class Account extends Profile {
  constructor(data) {
    super(data)
    this.email = data.email
  }
}

File: Dragon_Dungeons/Models/Account.cs
namespace Dragon_Dungeons.Models
{
  public class Profile : RepoItem
  {
    public string Name { get; set; }
    public string Picture { get; set; }
    public string CoverImg { get; set; }
  }

  public class Account : Profile
  {
    public string Email { get; set; }
  }
}